assets/railcraft/lang/zh_CN.lang

```diff
@@ -1,5 +1,5 @@
 # zh_CN
```

Comment:
Can this be moved to end of file to make alignment with `en_US.lang` better to make localization easier?